[2022-06-28 17:44:25,307] {processor.py:153} INFO - Started process (PID=9691) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:44:25,308] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:44:25,308] {logging_mixin.py:115} INFO - [2022-06-28 17:44:25,308] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:44:25,336] {logging_mixin.py:115} INFO - [2022-06-28 17:44:25,336] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:44:25,336] {logging_mixin.py:115} INFO - [2022-06-28 17:44:25,336] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:44:25,337] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:44:25,349] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2022-06-28 17:45:04,677] {processor.py:153} INFO - Started process (PID=9727) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:45:04,679] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:45:04,680] {logging_mixin.py:115} INFO - [2022-06-28 17:45:04,680] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:45:04,708] {logging_mixin.py:115} INFO - [2022-06-28 17:45:04,707] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:45:04,708] {logging_mixin.py:115} INFO - [2022-06-28 17:45:04,708] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:45:04,708] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:45:04,718] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2022-06-28 17:46:57,612] {processor.py:153} INFO - Started process (PID=9791) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:46:57,650] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:46:57,651] {logging_mixin.py:115} INFO - [2022-06-28 17:46:57,651] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:46:57,679] {logging_mixin.py:115} INFO - [2022-06-28 17:46:57,678] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:46:57,679] {logging_mixin.py:115} INFO - [2022-06-28 17:46:57,679] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:46:57,679] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:46:57,694] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2022-06-28 17:47:34,470] {processor.py:153} INFO - Started process (PID=9844) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:47:34,511] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:47:34,512] {logging_mixin.py:115} INFO - [2022-06-28 17:47:34,512] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:47:34,544] {logging_mixin.py:115} INFO - [2022-06-28 17:47:34,543] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:47:34,544] {logging_mixin.py:115} INFO - [2022-06-28 17:47:34,544] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:47:34,544] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:47:34,563] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.098 seconds
[2022-06-28 17:48:10,944] {processor.py:153} INFO - Started process (PID=10075) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:48:10,961] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:48:10,962] {logging_mixin.py:115} INFO - [2022-06-28 17:48:10,961] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:48:10,980] {logging_mixin.py:115} INFO - [2022-06-28 17:48:10,980] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:48:10,980] {logging_mixin.py:115} INFO - [2022-06-28 17:48:10,980] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:48:10,980] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:48:10,989] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-06-28 17:48:47,753] {processor.py:153} INFO - Started process (PID=10111) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:48:47,757] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:48:47,758] {logging_mixin.py:115} INFO - [2022-06-28 17:48:47,758] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:48:47,785] {logging_mixin.py:115} INFO - [2022-06-28 17:48:47,784] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:48:47,785] {logging_mixin.py:115} INFO - [2022-06-28 17:48:47,785] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:48:47,785] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:48:47,803] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-06-28 17:49:25,439] {processor.py:153} INFO - Started process (PID=10151) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:49:25,441] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:49:25,442] {logging_mixin.py:115} INFO - [2022-06-28 17:49:25,441] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:49:25,458] {logging_mixin.py:115} INFO - [2022-06-28 17:49:25,457] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:49:25,458] {logging_mixin.py:115} INFO - [2022-06-28 17:49:25,458] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:49:25,458] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:49:25,469] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2022-06-28 17:50:03,040] {processor.py:153} INFO - Started process (PID=10205) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:50:03,081] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:50:03,083] {logging_mixin.py:115} INFO - [2022-06-28 17:50:03,082] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:50:03,111] {logging_mixin.py:115} INFO - [2022-06-28 17:50:03,110] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:50:03,111] {logging_mixin.py:115} INFO - [2022-06-28 17:50:03,111] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:50:03,111] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:50:03,130] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2022-06-28 17:50:41,441] {processor.py:153} INFO - Started process (PID=10242) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:50:41,479] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:50:41,480] {logging_mixin.py:115} INFO - [2022-06-28 17:50:41,480] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:50:41,509] {logging_mixin.py:115} INFO - [2022-06-28 17:50:41,508] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:50:41,509] {logging_mixin.py:115} INFO - [2022-06-28 17:50:41,509] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:50:41,509] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:50:41,523] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2022-06-28 17:51:19,582] {processor.py:153} INFO - Started process (PID=10297) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:51:19,598] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:51:19,600] {logging_mixin.py:115} INFO - [2022-06-28 17:51:19,600] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:51:19,628] {logging_mixin.py:115} INFO - [2022-06-28 17:51:19,627] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:51:19,628] {logging_mixin.py:115} INFO - [2022-06-28 17:51:19,628] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:51:19,628] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:51:19,638] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2022-06-28 17:51:55,138] {processor.py:153} INFO - Started process (PID=10416) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:51:55,179] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:51:55,180] {logging_mixin.py:115} INFO - [2022-06-28 17:51:55,180] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:51:55,207] {logging_mixin.py:115} INFO - [2022-06-28 17:51:55,207] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:51:55,208] {logging_mixin.py:115} INFO - [2022-06-28 17:51:55,207] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:51:55,208] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:51:55,223] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2022-06-28 17:52:32,898] {processor.py:153} INFO - Started process (PID=10452) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:52:32,900] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:52:32,901] {logging_mixin.py:115} INFO - [2022-06-28 17:52:32,901] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:52:32,917] {logging_mixin.py:115} INFO - [2022-06-28 17:52:32,917] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:52:32,918] {logging_mixin.py:115} INFO - [2022-06-28 17:52:32,918] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:52:32,918] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:52:32,928] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2022-06-28 17:53:09,891] {processor.py:153} INFO - Started process (PID=10488) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:53:09,893] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:53:09,894] {logging_mixin.py:115} INFO - [2022-06-28 17:53:09,894] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:53:09,921] {logging_mixin.py:115} INFO - [2022-06-28 17:53:09,920] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:53:09,921] {logging_mixin.py:115} INFO - [2022-06-28 17:53:09,921] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:53:09,922] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:53:09,941] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-06-28 17:53:49,048] {processor.py:153} INFO - Started process (PID=10529) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:53:49,049] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:53:49,050] {logging_mixin.py:115} INFO - [2022-06-28 17:53:49,050] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:53:49,082] {logging_mixin.py:115} INFO - [2022-06-28 17:53:49,081] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:53:49,082] {logging_mixin.py:115} INFO - [2022-06-28 17:53:49,082] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:53:49,082] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:53:49,096] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.055 seconds
[2022-06-28 17:54:29,734] {processor.py:153} INFO - Started process (PID=10569) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:54:29,773] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:54:29,774] {logging_mixin.py:115} INFO - [2022-06-28 17:54:29,774] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:54:29,803] {logging_mixin.py:115} INFO - [2022-06-28 17:54:29,802] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:54:29,803] {logging_mixin.py:115} INFO - [2022-06-28 17:54:29,803] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:54:29,803] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:54:29,821] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2022-06-28 17:55:11,153] {processor.py:153} INFO - Started process (PID=10613) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:55:11,191] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:55:11,192] {logging_mixin.py:115} INFO - [2022-06-28 17:55:11,192] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:55:11,210] {logging_mixin.py:115} INFO - [2022-06-28 17:55:11,209] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:55:11,210] {logging_mixin.py:115} INFO - [2022-06-28 17:55:11,210] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:55:11,210] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:55:11,221] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2022-06-28 17:55:47,760] {processor.py:153} INFO - Started process (PID=10651) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:55:47,805] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:55:47,806] {logging_mixin.py:115} INFO - [2022-06-28 17:55:47,806] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:55:47,821] {logging_mixin.py:115} INFO - [2022-06-28 17:55:47,820] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:55:47,821] {logging_mixin.py:115} INFO - [2022-06-28 17:55:47,821] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:55:47,821] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:55:47,835] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2022-06-28 17:56:23,719] {processor.py:153} INFO - Started process (PID=10686) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:56:23,760] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:56:23,761] {logging_mixin.py:115} INFO - [2022-06-28 17:56:23,761] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:56:23,786] {logging_mixin.py:115} INFO - [2022-06-28 17:56:23,786] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:56:23,787] {logging_mixin.py:115} INFO - [2022-06-28 17:56:23,786] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:56:23,787] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:56:23,797] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.084 seconds
[2022-06-28 17:57:05,694] {processor.py:153} INFO - Started process (PID=10729) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:57:05,696] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:57:05,697] {logging_mixin.py:115} INFO - [2022-06-28 17:57:05,696] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:57:05,710] {logging_mixin.py:115} INFO - [2022-06-28 17:57:05,709] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:57:05,710] {logging_mixin.py:115} INFO - [2022-06-28 17:57:05,710] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:57:05,710] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:57:05,722] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2022-06-28 17:57:46,087] {processor.py:153} INFO - Started process (PID=10770) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:57:46,089] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:57:46,090] {logging_mixin.py:115} INFO - [2022-06-28 17:57:46,089] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:57:46,124] {logging_mixin.py:115} INFO - [2022-06-28 17:57:46,123] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:57:46,124] {logging_mixin.py:115} INFO - [2022-06-28 17:57:46,124] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:57:46,125] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:57:46,141] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2022-06-28 17:58:24,177] {processor.py:153} INFO - Started process (PID=10811) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:58:24,218] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:58:24,220] {logging_mixin.py:115} INFO - [2022-06-28 17:58:24,219] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:58:24,245] {logging_mixin.py:115} INFO - [2022-06-28 17:58:24,244] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:58:24,245] {logging_mixin.py:115} INFO - [2022-06-28 17:58:24,245] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:58:24,245] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:58:24,262] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2022-06-28 17:59:02,795] {processor.py:153} INFO - Started process (PID=10850) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:59:02,824] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:59:02,825] {logging_mixin.py:115} INFO - [2022-06-28 17:59:02,825] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:59:02,854] {logging_mixin.py:115} INFO - [2022-06-28 17:59:02,853] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:59:02,854] {logging_mixin.py:115} INFO - [2022-06-28 17:59:02,854] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:59:02,854] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:59:02,871] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2022-06-28 17:59:40,430] {processor.py:153} INFO - Started process (PID=10886) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:59:40,470] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 17:59:40,471] {logging_mixin.py:115} INFO - [2022-06-28 17:59:40,471] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:59:40,498] {logging_mixin.py:115} INFO - [2022-06-28 17:59:40,498] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 17:59:40,498] {logging_mixin.py:115} INFO - [2022-06-28 17:59:40,498] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 17:59:40,499] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 17:59:40,512] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2022-06-28 18:00:16,555] {processor.py:153} INFO - Started process (PID=10925) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:00:16,558] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:00:16,559] {logging_mixin.py:115} INFO - [2022-06-28 18:00:16,559] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:00:16,589] {logging_mixin.py:115} INFO - [2022-06-28 18:00:16,589] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:00:16,590] {logging_mixin.py:115} INFO - [2022-06-28 18:00:16,589] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:00:16,590] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:00:16,608] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2022-06-28 18:00:54,642] {processor.py:153} INFO - Started process (PID=10962) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:00:54,683] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:00:54,684] {logging_mixin.py:115} INFO - [2022-06-28 18:00:54,684] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:00:54,713] {logging_mixin.py:115} INFO - [2022-06-28 18:00:54,712] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:00:54,713] {logging_mixin.py:115} INFO - [2022-06-28 18:00:54,713] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:00:54,714] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:00:54,727] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2022-06-28 18:01:31,183] {processor.py:153} INFO - Started process (PID=11000) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:01:31,192] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:01:31,193] {logging_mixin.py:115} INFO - [2022-06-28 18:01:31,193] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:01:31,224] {logging_mixin.py:115} INFO - [2022-06-28 18:01:31,224] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:01:31,225] {logging_mixin.py:115} INFO - [2022-06-28 18:01:31,225] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:01:31,225] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:01:31,238] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2022-06-28 18:02:12,730] {processor.py:153} INFO - Started process (PID=11042) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:02:12,732] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:02:12,733] {logging_mixin.py:115} INFO - [2022-06-28 18:02:12,733] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:02:12,766] {logging_mixin.py:115} INFO - [2022-06-28 18:02:12,765] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:02:12,766] {logging_mixin.py:115} INFO - [2022-06-28 18:02:12,766] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:02:12,766] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:02:12,783] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2022-06-28 18:02:50,580] {processor.py:153} INFO - Started process (PID=11078) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:02:50,582] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:02:50,582] {logging_mixin.py:115} INFO - [2022-06-28 18:02:50,582] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:02:50,599] {logging_mixin.py:115} INFO - [2022-06-28 18:02:50,599] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:02:50,600] {logging_mixin.py:115} INFO - [2022-06-28 18:02:50,600] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:02:50,600] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:02:50,608] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2022-06-28 18:03:27,030] {processor.py:153} INFO - Started process (PID=11114) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:03:27,033] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:03:27,034] {logging_mixin.py:115} INFO - [2022-06-28 18:03:27,034] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:03:27,058] {logging_mixin.py:115} INFO - [2022-06-28 18:03:27,058] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:03:27,059] {logging_mixin.py:115} INFO - [2022-06-28 18:03:27,059] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:03:27,059] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:03:27,073] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2022-06-28 18:04:03,138] {processor.py:153} INFO - Started process (PID=11148) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:04:03,179] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:04:03,180] {logging_mixin.py:115} INFO - [2022-06-28 18:04:03,179] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:04:03,207] {logging_mixin.py:115} INFO - [2022-06-28 18:04:03,206] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:04:03,207] {logging_mixin.py:115} INFO - [2022-06-28 18:04:03,207] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:04:03,208] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:04:03,224] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2022-06-28 18:04:43,695] {processor.py:153} INFO - Started process (PID=11187) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:04:43,736] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:04:43,737] {logging_mixin.py:115} INFO - [2022-06-28 18:04:43,737] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:04:43,765] {logging_mixin.py:115} INFO - [2022-06-28 18:04:43,765] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:04:43,766] {logging_mixin.py:115} INFO - [2022-06-28 18:04:43,765] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:04:43,766] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:04:43,782] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.094 seconds
[2022-06-28 18:05:18,016] {processor.py:153} INFO - Started process (PID=11221) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:05:18,017] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:05:18,018] {logging_mixin.py:115} INFO - [2022-06-28 18:05:18,018] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:05:18,041] {logging_mixin.py:115} INFO - [2022-06-28 18:05:18,040] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:05:18,041] {logging_mixin.py:115} INFO - [2022-06-28 18:05:18,041] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:05:18,041] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:05:18,059] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2022-06-28 18:05:53,618] {processor.py:153} INFO - Started process (PID=11256) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:05:53,620] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:05:53,621] {logging_mixin.py:115} INFO - [2022-06-28 18:05:53,621] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:05:53,648] {logging_mixin.py:115} INFO - [2022-06-28 18:05:53,647] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:05:53,648] {logging_mixin.py:115} INFO - [2022-06-28 18:05:53,648] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:05:53,648] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:05:53,666] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2022-06-28 18:06:35,451] {processor.py:153} INFO - Started process (PID=11294) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:06:35,486] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:06:35,487] {logging_mixin.py:115} INFO - [2022-06-28 18:06:35,487] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:06:35,515] {logging_mixin.py:115} INFO - [2022-06-28 18:06:35,514] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:06:35,515] {logging_mixin.py:115} INFO - [2022-06-28 18:06:35,515] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:06:35,515] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:06:35,533] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2022-06-28 18:07:12,062] {processor.py:153} INFO - Started process (PID=11330) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:07:12,102] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:07:12,104] {logging_mixin.py:115} INFO - [2022-06-28 18:07:12,103] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:07:12,131] {logging_mixin.py:115} INFO - [2022-06-28 18:07:12,131] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:07:12,131] {logging_mixin.py:115} INFO - [2022-06-28 18:07:12,131] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:07:12,132] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:07:12,144] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.088 seconds
[2022-06-28 18:07:49,029] {processor.py:153} INFO - Started process (PID=11374) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:07:49,083] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:07:49,085] {logging_mixin.py:115} INFO - [2022-06-28 18:07:49,084] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:07:49,107] {logging_mixin.py:115} INFO - [2022-06-28 18:07:49,106] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:07:49,107] {logging_mixin.py:115} INFO - [2022-06-28 18:07:49,107] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:07:49,107] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:07:49,119] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2022-06-28 18:08:39,199] {processor.py:153} INFO - Started process (PID=11428) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:08:39,201] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:08:39,202] {logging_mixin.py:115} INFO - [2022-06-28 18:08:39,201] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:08:39,223] {logging_mixin.py:115} INFO - [2022-06-28 18:08:39,222] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:08:39,223] {logging_mixin.py:115} INFO - [2022-06-28 18:08:39,223] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:08:39,223] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:08:39,239] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2022-06-28 18:09:22,606] {processor.py:153} INFO - Started process (PID=11681) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:09:22,608] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:09:22,608] {logging_mixin.py:115} INFO - [2022-06-28 18:09:22,608] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:09:22,630] {logging_mixin.py:115} INFO - [2022-06-28 18:09:22,629] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:09:22,631] {logging_mixin.py:115} INFO - [2022-06-28 18:09:22,630] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:09:22,631] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:09:22,644] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2022-06-28 18:10:02,487] {processor.py:153} INFO - Started process (PID=11724) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:10:02,488] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:10:02,489] {logging_mixin.py:115} INFO - [2022-06-28 18:10:02,489] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:10:02,512] {logging_mixin.py:115} INFO - [2022-06-28 18:10:02,511] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:10:02,512] {logging_mixin.py:115} INFO - [2022-06-28 18:10:02,512] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:10:02,512] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:10:02,524] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2022-06-28 18:11:17,046] {processor.py:153} INFO - Started process (PID=11826) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:11:17,084] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:11:17,085] {logging_mixin.py:115} INFO - [2022-06-28 18:11:17,085] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:11:17,111] {logging_mixin.py:115} INFO - [2022-06-28 18:11:17,111] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:11:17,111] {logging_mixin.py:115} INFO - [2022-06-28 18:11:17,111] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:11:17,111] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:11:17,123] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2022-06-28 18:12:32,705] {processor.py:153} INFO - Started process (PID=11925) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:12:32,707] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:12:32,709] {logging_mixin.py:115} INFO - [2022-06-28 18:12:32,709] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:12:32,738] {logging_mixin.py:115} INFO - [2022-06-28 18:12:32,737] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:12:32,738] {logging_mixin.py:115} INFO - [2022-06-28 18:12:32,738] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:12:32,738] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:12:32,781] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2022-06-28 18:14:42,929] {processor.py:153} INFO - Started process (PID=12748) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:14:42,931] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:14:42,932] {logging_mixin.py:115} INFO - [2022-06-28 18:14:42,931] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:14:42,946] {logging_mixin.py:115} INFO - [2022-06-28 18:14:42,946] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:14:42,947] {logging_mixin.py:115} INFO - [2022-06-28 18:14:42,947] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:14:42,947] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:14:42,955] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2022-06-28 18:17:03,044] {processor.py:153} INFO - Started process (PID=13545) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:17:03,098] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:17:03,100] {logging_mixin.py:115} INFO - [2022-06-28 18:17:03,099] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:17:03,126] {logging_mixin.py:115} INFO - [2022-06-28 18:17:03,126] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:17:03,127] {logging_mixin.py:115} INFO - [2022-06-28 18:17:03,127] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:17:03,127] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:17:03,145] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.108 seconds
[2022-06-28 18:17:42,795] {processor.py:153} INFO - Started process (PID=13618) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:17:42,836] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:17:42,837] {logging_mixin.py:115} INFO - [2022-06-28 18:17:42,837] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:17:42,859] {logging_mixin.py:115} INFO - [2022-06-28 18:17:42,858] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:17:42,859] {logging_mixin.py:115} INFO - [2022-06-28 18:17:42,859] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:17:42,859] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:17:42,871] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2022-06-28 18:20:22,661] {processor.py:153} INFO - Started process (PID=14890) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:20:22,663] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:20:22,664] {logging_mixin.py:115} INFO - [2022-06-28 18:20:22,664] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:20:22,680] {logging_mixin.py:115} INFO - [2022-06-28 18:20:22,679] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:20:22,680] {logging_mixin.py:115} INFO - [2022-06-28 18:20:22,680] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:20:22,680] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:20:22,691] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2022-06-28 18:21:07,559] {processor.py:153} INFO - Started process (PID=14983) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:21:07,590] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:21:07,591] {logging_mixin.py:115} INFO - [2022-06-28 18:21:07,591] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:21:07,621] {logging_mixin.py:115} INFO - [2022-06-28 18:21:07,620] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:21:07,621] {logging_mixin.py:115} INFO - [2022-06-28 18:21:07,621] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:21:07,621] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:21:07,639] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2022-06-28 18:21:48,511] {processor.py:153} INFO - Started process (PID=15024) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:21:48,513] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:21:48,515] {logging_mixin.py:115} INFO - [2022-06-28 18:21:48,514] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:21:48,543] {logging_mixin.py:115} INFO - [2022-06-28 18:21:48,542] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:21:48,543] {logging_mixin.py:115} INFO - [2022-06-28 18:21:48,543] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:21:48,543] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:21:48,551] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2022-06-28 18:22:32,473] {processor.py:153} INFO - Started process (PID=15112) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:22:32,514] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:22:32,516] {logging_mixin.py:115} INFO - [2022-06-28 18:22:32,515] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:22:32,546] {logging_mixin.py:115} INFO - [2022-06-28 18:22:32,546] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:22:32,547] {logging_mixin.py:115} INFO - [2022-06-28 18:22:32,547] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:22:32,547] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:22:32,567] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2022-06-28 18:23:22,935] {processor.py:153} INFO - Started process (PID=15228) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:23:22,937] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:23:22,938] {logging_mixin.py:115} INFO - [2022-06-28 18:23:22,937] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:23:22,958] {logging_mixin.py:115} INFO - [2022-06-28 18:23:22,957] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:23:22,958] {logging_mixin.py:115} INFO - [2022-06-28 18:23:22,958] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:23:22,958] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:23:22,968] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2022-06-28 18:24:12,483] {processor.py:153} INFO - Started process (PID=15354) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:24:12,484] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:24:12,485] {logging_mixin.py:115} INFO - [2022-06-28 18:24:12,485] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:24:12,513] {logging_mixin.py:115} INFO - [2022-06-28 18:24:12,513] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:24:12,514] {logging_mixin.py:115} INFO - [2022-06-28 18:24:12,513] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:24:12,514] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:24:12,528] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2022-06-28 18:24:51,834] {processor.py:153} INFO - Started process (PID=15391) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:24:51,874] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:24:51,875] {logging_mixin.py:115} INFO - [2022-06-28 18:24:51,875] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:24:51,904] {logging_mixin.py:115} INFO - [2022-06-28 18:24:51,904] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:24:51,905] {logging_mixin.py:115} INFO - [2022-06-28 18:24:51,905] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:24:51,905] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:24:51,923] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.097 seconds
[2022-06-28 18:25:35,754] {processor.py:153} INFO - Started process (PID=15465) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:25:35,795] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:25:35,796] {logging_mixin.py:115} INFO - [2022-06-28 18:25:35,796] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:25:35,827] {logging_mixin.py:115} INFO - [2022-06-28 18:25:35,826] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:25:35,827] {logging_mixin.py:115} INFO - [2022-06-28 18:25:35,827] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:25:35,827] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:25:35,847] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2022-06-28 18:26:22,516] {processor.py:153} INFO - Started process (PID=15545) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:26:22,556] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:26:22,557] {logging_mixin.py:115} INFO - [2022-06-28 18:26:22,557] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:26:22,574] {logging_mixin.py:115} INFO - [2022-06-28 18:26:22,573] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:26:22,574] {logging_mixin.py:115} INFO - [2022-06-28 18:26:22,574] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:26:22,574] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:26:22,584] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2022-06-28 18:27:16,062] {processor.py:153} INFO - Started process (PID=15627) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:27:16,065] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:27:16,066] {logging_mixin.py:115} INFO - [2022-06-28 18:27:16,066] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:27:16,093] {logging_mixin.py:115} INFO - [2022-06-28 18:27:16,092] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:27:16,093] {logging_mixin.py:115} INFO - [2022-06-28 18:27:16,093] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:27:16,093] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:27:16,107] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2022-06-28 18:28:09,009] {processor.py:153} INFO - Started process (PID=15709) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:28:09,028] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:28:09,029] {logging_mixin.py:115} INFO - [2022-06-28 18:28:09,029] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:28:09,055] {logging_mixin.py:115} INFO - [2022-06-28 18:28:09,054] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:28:09,055] {logging_mixin.py:115} INFO - [2022-06-28 18:28:09,055] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:28:09,055] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:28:09,073] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2022-06-28 18:28:48,818] {processor.py:153} INFO - Started process (PID=15751) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:28:48,859] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:28:48,861] {logging_mixin.py:115} INFO - [2022-06-28 18:28:48,860] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:28:48,888] {logging_mixin.py:115} INFO - [2022-06-28 18:28:48,888] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:28:48,889] {logging_mixin.py:115} INFO - [2022-06-28 18:28:48,889] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:28:48,889] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:28:48,908] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2022-06-28 18:29:33,195] {processor.py:153} INFO - Started process (PID=15835) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:29:33,198] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:29:33,199] {logging_mixin.py:115} INFO - [2022-06-28 18:29:33,199] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:29:33,224] {logging_mixin.py:115} INFO - [2022-06-28 18:29:33,223] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:29:33,224] {logging_mixin.py:115} INFO - [2022-06-28 18:29:33,224] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:29:33,224] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:29:33,240] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2022-06-28 18:30:25,775] {processor.py:153} INFO - Started process (PID=15952) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:30:25,777] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:30:25,777] {logging_mixin.py:115} INFO - [2022-06-28 18:30:25,777] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:30:25,809] {logging_mixin.py:115} INFO - [2022-06-28 18:30:25,808] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:30:25,810] {logging_mixin.py:115} INFO - [2022-06-28 18:30:25,809] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:30:25,810] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:30:25,832] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2022-06-28 18:31:11,781] {processor.py:153} INFO - Started process (PID=16042) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:31:11,822] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:31:11,823] {logging_mixin.py:115} INFO - [2022-06-28 18:31:11,823] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:31:11,851] {logging_mixin.py:115} INFO - [2022-06-28 18:31:11,850] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:31:11,851] {logging_mixin.py:115} INFO - [2022-06-28 18:31:11,851] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:31:11,851] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:31:11,867] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2022-06-28 18:32:00,740] {processor.py:153} INFO - Started process (PID=16123) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:32:00,741] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:32:00,741] {logging_mixin.py:115} INFO - [2022-06-28 18:32:00,741] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:32:00,758] {logging_mixin.py:115} INFO - [2022-06-28 18:32:00,758] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:32:00,758] {logging_mixin.py:115} INFO - [2022-06-28 18:32:00,758] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:32:00,758] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:32:00,769] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2022-06-28 18:32:41,002] {processor.py:153} INFO - Started process (PID=17979) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:32:41,004] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:32:41,005] {logging_mixin.py:115} INFO - [2022-06-28 18:32:41,005] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:32:41,033] {logging_mixin.py:115} INFO - [2022-06-28 18:32:41,032] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:32:41,033] {logging_mixin.py:115} INFO - [2022-06-28 18:32:41,033] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:32:41,033] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:32:41,045] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2022-06-28 18:33:31,016] {processor.py:153} INFO - Started process (PID=20806) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:33:31,017] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:33:31,017] {logging_mixin.py:115} INFO - [2022-06-28 18:33:31,017] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:33:31,029] {logging_mixin.py:115} INFO - [2022-06-28 18:33:31,028] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:33:31,029] {logging_mixin.py:115} INFO - [2022-06-28 18:33:31,029] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:33:31,029] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:33:31,038] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2022-06-28 18:34:20,572] {processor.py:153} INFO - Started process (PID=20893) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:34:20,575] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:34:20,575] {logging_mixin.py:115} INFO - [2022-06-28 18:34:20,575] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:34:20,590] {logging_mixin.py:115} INFO - [2022-06-28 18:34:20,589] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:34:20,590] {logging_mixin.py:115} INFO - [2022-06-28 18:34:20,590] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:34:20,590] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:34:20,603] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2022-06-28 18:35:12,016] {processor.py:153} INFO - Started process (PID=20975) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:35:12,017] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:35:12,018] {logging_mixin.py:115} INFO - [2022-06-28 18:35:12,018] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:35:12,049] {logging_mixin.py:115} INFO - [2022-06-28 18:35:12,048] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:35:12,050] {logging_mixin.py:115} INFO - [2022-06-28 18:35:12,050] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:35:12,050] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:35:12,064] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2022-06-28 18:35:57,386] {processor.py:153} INFO - Started process (PID=21051) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:35:57,392] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:35:57,393] {logging_mixin.py:115} INFO - [2022-06-28 18:35:57,393] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:35:57,421] {logging_mixin.py:115} INFO - [2022-06-28 18:35:57,420] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:35:57,421] {logging_mixin.py:115} INFO - [2022-06-28 18:35:57,421] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:35:57,422] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:35:57,436] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.056 seconds
[2022-06-28 18:36:37,906] {processor.py:153} INFO - Started process (PID=21088) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:36:37,939] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:36:37,941] {logging_mixin.py:115} INFO - [2022-06-28 18:36:37,941] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:36:37,971] {logging_mixin.py:115} INFO - [2022-06-28 18:36:37,970] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:36:37,971] {logging_mixin.py:115} INFO - [2022-06-28 18:36:37,971] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:36:37,972] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:36:37,989] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.090 seconds
[2022-06-28 18:37:21,641] {processor.py:153} INFO - Started process (PID=21164) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:37:21,642] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:37:21,642] {logging_mixin.py:115} INFO - [2022-06-28 18:37:21,642] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:37:21,659] {logging_mixin.py:115} INFO - [2022-06-28 18:37:21,659] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:37:21,659] {logging_mixin.py:115} INFO - [2022-06-28 18:37:21,659] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:37:21,659] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:37:21,668] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2022-06-28 18:38:12,754] {processor.py:153} INFO - Started process (PID=21245) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:38:12,794] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:38:12,795] {logging_mixin.py:115} INFO - [2022-06-28 18:38:12,795] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:38:12,816] {logging_mixin.py:115} INFO - [2022-06-28 18:38:12,816] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:38:12,817] {logging_mixin.py:115} INFO - [2022-06-28 18:38:12,817] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:38:12,817] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:38:12,833] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2022-06-28 18:38:58,824] {processor.py:153} INFO - Started process (PID=21323) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:38:58,837] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:38:58,838] {logging_mixin.py:115} INFO - [2022-06-28 18:38:58,838] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:38:58,868] {logging_mixin.py:115} INFO - [2022-06-28 18:38:58,868] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:38:58,868] {logging_mixin.py:115} INFO - [2022-06-28 18:38:58,868] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:38:58,869] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:38:58,887] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2022-06-28 18:39:44,377] {processor.py:153} INFO - Started process (PID=21367) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:39:44,417] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:39:44,419] {logging_mixin.py:115} INFO - [2022-06-28 18:39:44,419] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:39:44,449] {logging_mixin.py:115} INFO - [2022-06-28 18:39:44,448] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:39:44,449] {logging_mixin.py:115} INFO - [2022-06-28 18:39:44,449] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:39:44,450] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:39:44,467] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2022-06-28 18:40:28,479] {processor.py:153} INFO - Started process (PID=21441) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:40:28,481] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:40:28,481] {logging_mixin.py:115} INFO - [2022-06-28 18:40:28,481] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:40:28,511] {logging_mixin.py:115} INFO - [2022-06-28 18:40:28,510] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:40:28,511] {logging_mixin.py:115} INFO - [2022-06-28 18:40:28,511] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:40:28,512] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:40:28,523] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2022-06-28 18:41:19,820] {processor.py:153} INFO - Started process (PID=21521) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:41:19,823] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:41:19,824] {logging_mixin.py:115} INFO - [2022-06-28 18:41:19,824] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:41:19,852] {logging_mixin.py:115} INFO - [2022-06-28 18:41:19,851] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:41:19,852] {logging_mixin.py:115} INFO - [2022-06-28 18:41:19,852] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:41:19,853] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:41:19,871] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2022-06-28 18:42:16,197] {processor.py:153} INFO - Started process (PID=21606) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:42:16,238] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:42:16,239] {logging_mixin.py:115} INFO - [2022-06-28 18:42:16,239] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:42:16,269] {logging_mixin.py:115} INFO - [2022-06-28 18:42:16,268] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:42:16,269] {logging_mixin.py:115} INFO - [2022-06-28 18:42:16,269] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:42:16,269] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:42:16,280] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2022-06-28 18:43:06,716] {processor.py:153} INFO - Started process (PID=21687) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:43:06,756] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:43:06,758] {logging_mixin.py:115} INFO - [2022-06-28 18:43:06,757] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:43:06,787] {logging_mixin.py:115} INFO - [2022-06-28 18:43:06,786] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:43:06,787] {logging_mixin.py:115} INFO - [2022-06-28 18:43:06,787] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:43:06,787] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:43:06,796] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2022-06-28 18:43:47,267] {processor.py:153} INFO - Started process (PID=21726) to work on /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:43:47,268] {processor.py:641} INFO - Processing file /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-28 18:43:47,269] {logging_mixin.py:115} INFO - [2022-06-28 18:43:47,269] {dagbag.py:507} INFO - Filling up the DagBag from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:43:47,283] {logging_mixin.py:115} INFO - [2022-06-28 18:43:47,283] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-28 18:43:47,283] {logging_mixin.py:115} INFO - [2022-06-28 18:43:47,283] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-28 18:43:47,283] {processor.py:653} WARNING - No viable dags retrieved from /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-28 18:43:47,292] {processor.py:161} INFO - Processing /root/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
